multidb/tests.py

```diff
@@ -160,7 +161,7 @@ def write_view(req):
         assert PINNING_COOKIE in response.cookies
 
 
-class ContextDecoratorTests(TestCase):
+class UseMasterTests(TestCase):
```

Comment:
it's fine, but in general, extraneous changes just make reviewing pull reqs more difficult, i try to avoid them.